jdaviz/configs/cubeviz/plugins/slice/slice.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+  <j-tray-plugin>
+    <v-row>
+      <j-docs-link :link="'https://jdaviz.readthedocs.io/en/'+vdocs+'/'+config+'/plugins.html#slice'">Select slice (or wavelength) of the cube to show in the image viewers and highlighted in the spectrum viewer.  The slice can also be changed interactively in the spectrum viewer by activating the slice tool.</j-docs-link>
+    </v-row>
+
+    <v-row>
+      <v-expansion-panels popout>
+        <v-expansion-panel>
+          <v-expansion-panel-header v-slot="{ open }">
+            <span style="padding: 6px">Indicator Settings</span>
```

Comment:
Not sure if this helps or hurts but it is more informational for me.
```suggestion
            <span style="padding: 6px">Preferences</span>
```

Reply:
I personally prefer the context of keeping "Indicator" (otherwise I think I'd expect app-wide or at least plugin-wide options), but have no inclination towards "Settings" vs "Preferences".  That said - if we want to start using similar UI across other plugins, maybe plugin-wide settings/preferences makes sense when applicable.  Anyone else have opinions/votes?